excellent/legacy_test.go

```diff
@@ -17,12 +17,12 @@ func TestTranslate(t *testing.T) {
 		{old: "@contact", new: "@contact"},
 		{old: "@contact.first_name", new: "@contact.first_name"},
 		{old: "@contact.name", new: "@contact.name"},
-		{old: "@contact.tel", new: "@contact.urns.tel"},
-		{old: "@contact.tel_e164", new: "@contact.urns.tel_e164"},
-		{old: "@contact.telegram", new: "@contact.urns.telegram"},
-		{old: "@contact.twitter", new: "@contact.urns.twitter"},
-		{old: "@contact.facebook", new: "@contact.urns.facebook"},
-		{old: "@contact.mailto", new: "@contact.urns.mailto"},
+		{old: "@contact.tel", new: "@(format_urn(contact.urns.tel.0))"},
+		{old: "@contact.tel_e164", new: "@contact.urns.tel.0.path"},
+		{old: "@contact.telegram", new: "@(format_urn(contact.urns.telegram.0))"},
+		{old: "@contact.twitter", new: "@(format_urn(contact.urns.twitter.0))"},
+		{old: "@contact.facebook", new: "@(format_urn(contact.urns.facebook.0))"},
+		{old: "@contact.mailto", new: "@(format_urn(contact.urns.mailto.0))"},
```

Comment:
Do we need the `.0`s here?

Reply:
No but I think better the users see that these are lists and we treat passing a list to `format_urn` as a convenience.

Reply:
Hah, funny, feels the opposite to me. Since 99% of users will never have more than one URN, present them with the simplest option that works.

Reply:
Well I guess I'd be ok with making this look simpler. I actually wonder tho how common `format_urn` usage is going to be since I imagine most of the time people want to reference contact URNs, it's not to display them in a message, but to pass them to a webhook or something like that